R/uvhydrograph-data.R

```diff
@@ -19,6 +19,14 @@ parseUVData <- function(data, plotName, month) {
     
     series_corr <- subsetByMonth(getCorrections(data, "primarySeriesCorrections"), month)
     meas_Q <- subsetByMonth(getFieldVisitMeasurementsQPoints(data), month)  
+
+    #Add reference data to the plot if it is available and this is a Q plot type
+    if(any(grepl("Discharge", getReportMetadata(data,'primaryParameter'))))
+    {
+      #Reference Time Series Data
+      corr_UV_Qref <- subsetByMonth(getTimeSeries(data, "downsampledReferenceSeries"), month)
+      est_UV_Qref <- subsetByMonth(getTimeSeries(data, "downsampledReferenceSeries", estimatedOnly=TRUE), month)
+    }
```

Comment:
Note: Moving this block up was to fix an unrelated issue of the reference TS plotting on top of the approval points.